src/interp.c

```diff
@@ -429,9 +429,8 @@ static char *apply_prefix(char *buf, CHAR_DATA *ch, char *command) {
         return command;
     } else {
         int prefix_len = strlen(pc_prefix);
-        memcpy(buf, pc_prefix, (prefix_len < MAX_INPUT_LENGTH) ? prefix_len : MAX_INPUT_LENGTH - 1);
-        memcpy(buf + prefix_len, command, MAX_INPUT_LENGTH - prefix_len);
```

Comment:
In debug builds, it'd bomb out on line 433 with a complaint about a stack overflow, and that cmd_buf underflows the "command" variable:

```
  This frame has 3 object(s):
    [32, 288) 'cmd_buf'
    [320, 576) 'command' <== Memory access at offset 288 partially underflows this variable
    [608, 864) 'logline'


```

Reply:
I found that zeroing out the cmd_buf and using strncpy & strncat seemed to resolve it.  Thoughts?

Reply:
Note: the 'at' command is a bit special as it parses its first arg (name of target character),  finds that, then sends the rest of your command string to interpret() to be interpreted like normal input.

Reply:
Alright it seems that using strncpy() is itself nearly as bad, and won't compile on the newer compiler due to the size being copied is the same as the string length.

Reply:
yikes yeah... this is super dodgy code (as it was before that is). I'm still thinking...